deeppavlov/contrib/skills/similarity_matching_skill/similarity_matching_skill.py

```diff
@@ -77,6 +82,15 @@ def __call__(self, utterances_batch: List[str], history_batch: List[List[str]],
                 each utterance.
 
         Returns:
-            Batches of skill inference results and estimated confidences
+            Batches of the skill inference results and estimated confidences.
         """
-        return self.model(utterances_batch)
+        model_result = self.model(utterances_batch)
+
+        responses = [r for r in model_result[0]]
+        confidences = [r for r in model_result[1]]
```

Comment:
```python3
confidences = model_result[1]
```
or if you want the exact result
```python3
confidences = list(model_result[1])
```
but I would recommend
```python3
responses, confidences = self.model(utterances_batch)
```

Reply:
applied